packages/app-accounts/src/Account.tsx

```diff
@@ -110,7 +113,8 @@ class Account extends React.PureComponent<Props> {
     this.state = {
       isBackupOpen: false,
       isForgetOpen: false,
-      isPasswordOpen: false
+      isPasswordOpen: false,
+      isTransferOpen: false
```

Comment:
As a first step, think this is fine - but we probably need to give some thought of moving it elsewhere so it is re-usable. (Especially when we get to deposit on addresses - but for now, 100% - let's not try to complicate things, moving code is easy, getting it working initially is the hard bit)